android/src/main/java/com/jaumard/smsautofill/SmsAutoFillPlugin.java

```diff
@@ -111,6 +116,12 @@ public void onFailure(@NonNull Exception e) {
                 pendingHintResult = result;
                 requestHint();
                 break;
+            case "getAppSignature":
+                AppSignatureHelper signatureHelper = new AppSignatureHelper(registrar.context());
+                ArrayList<String> appSignatures = signatureHelper.getAppSignatures();
+                String signature = appSignatures.get(0);
```

Comment:
this will crash if no signature is returned, if we need only one `getAppSignatures` should return a String directly